packages/invoice-dashboard/src/utils/getConversionPaymentValues.ts

```
@@ -0,0 +1,168 @@
+export const bigAmountify = (
+  amount: number,
+  token: Pick<CurrencyTypes.CurrencyDefinition, "decimals">,
+): BigNumber => {
+  let [whole, decimals] = amount.toString().split(".");
+  let pow = "0";
+  let powSign = true;
+
+  if (decimals && decimals.includes("e")) {
+    powSign = !decimals.includes("e-");
+    [decimals, pow] = powSign ? decimals.split("e") : decimals.split("e-");
+    whole = whole;
```

Biome

[error] whole is assigned to itself.

This is where is assigned.


(lint/correctness/noSelfAssign)